card.go

```diff
@@ -163,6 +175,12 @@ func get(ix any, k string) any {
 func (c *Card) dump() CardD {
 	data := make(map[string]any)
 	var bufs []BufD
+
+	// Look for nested buffers within form_card.
+	if v, ok := c.data["view"]; ok && v.(string) == "form" {
```

Comment:
`v.(string)` might not be a string and can cause some trouble here. I'd handle it inside the `if` scope.

Reply:
I guess I need to write a nested if with type cast check, right? Or is there a way how to put it to the existing if statement?

Reply:
yes, nested if